docker-compose.yaml

```diff
@@ -11,3 +11,17 @@ services:
     environment:
       RABBITMQ_DEFAULT_USER: wanda
       RABBITMQ_DEFAULT_PASS: wanda
+  postgres:
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PGDATA: /var/lib/postgresql/data/pgdata
+    ports:
+      - 5433:5432
```

Comment:
we could change the port we forward so it doesn't clash with trento web, what about `5434` ?